getFeatureFlagById/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"log"
+	"net/http"
+)
```

Comment:
Please organize the imports in groups with blank lines between them.

Reference - https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Does this has any benefit or for the readability? @shubham-y

Reply:
If not can we handle it later?

Reply:
> Does this has any benefit or for the readability? @shubham-y 

Yes it's for better readability. Making this change in this PR or afterwards is your call. I am fine with both

Reply:
Will create a ticket for rearranging the imports.

- [ ] Rearrange the imports for better readability.

Reply:
https://github.com/Real-Dev-Squad/feature-flag-backend/issues/35

Reply:
Resolving this @shubham-y for now